tests/unit/run-task-test.js

```diff
@@ -1,14 +1,7 @@
 import EmberObject from '@ember/object';
 import { run } from '@ember/runloop';
+import { cancelTask, runDisposables, runTask, scheduleTask, throttleTask, _setRegisteredTimers } from 'ember-lifeline';
```

Comment:
I think you aren't using Prettier. We should reformat this to adhere to our Prettier config.